src/components/ReportActionItem/IOUPreview.js

```diff
@@ -128,14 +134,14 @@ const IOUPreview = (props) => {
     if (_.isEmpty(props.iouReport) && !props.isBillSplit) {
         return null;
     }
-    const sessionEmail = lodashGet(props.session, 'email', null);
-    const managerEmail = props.iouReport.managerEmail || '';
-    const ownerEmail = props.iouReport.ownerEmail || '';
-    const participantEmails = props.isBillSplit ? lodashGet(props.action, 'originalMessage.participants', []) : [managerEmail, ownerEmail];
-    const participantAvatars = OptionsListUtils.getAvatarsForLogins(participantEmails, props.personalDetails);
+    const sessionAccountID = lodashGet(props.session, 'accountID', null);
+    const managerID = props.iouReport.managerID || '';
+    const ownerAccountID = props.iouReport.ownerAccountID || '';
+    const participantAccountIDs = props.isBillSplit ? lodashGet(props.action, 'originalMessage.participantAccountIDs', []) : [managerID, ownerAccountID];
```

Comment:
We should have sorted the account ids. Not doing so resulted in https://github.com/Expensify/App/issues/31732